libnd4j/include/ops/declarable/helpers/cuda/concat.cu

```diff
@@ -31,38 +31,42 @@
 
 #include <numeric>
 
+#include "../../../../../../../../../../../usr/include/complex.h"
```

Comment:
I guess you need to properly reference this file

Reply:
Good catch...this was probably an auto import..